dissect/target/plugins/os/windows/dpapi/dpapi.py

```diff
@@ -1,162 +1,167 @@
-import hashlib
 import re
 from functools import cache, cached_property
 from pathlib import Path
-
-try:
-    from Crypto.Cipher import AES
-
-    HAS_CRYPTO = True
-except ImportError:
-    HAS_CRYPTO = False
-
+from typing import Iterator
 
 from dissect.target.exceptions import UnsupportedPluginError
-from dissect.target.helpers import keychain
 from dissect.target.plugin import InternalPlugin
 from dissect.target.plugins.os.windows.dpapi.blob import Blob as DPAPIBlob
 from dissect.target.plugins.os.windows.dpapi.master_key import CredSystem, MasterKeyFile
 from dissect.target.target import Target
 
 
 class DPAPIPlugin(InternalPlugin):
-    __namespace__ = "dpapi"
+    """Windows Data Protection API (DPAPI) plugin.
 
-    MASTER_KEY_REGEX = re.compile("^[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12}$")
+    Resources:
+        - Reversing ``Crypt32.dll``
+        - https://learn.microsoft.com/en-us/windows/win32/api/dpapi/
+        - https://github.com/fortra/impacket/blob/master/examples/dpapi.py
+        - https://github.com/tijldeneut/DPAPIck3
+        - https://www.passcape.com/index.php?section=docsys&cmd=details&id=28
+    """
 
-    SECURITY_POLICY_KEY = "HKEY_LOCAL_MACHINE\\SECURITY\\Policy"
-    SYSTEM_KEY = "HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\LSA"
+    __namespace__ = "dpapi"
 
+    MASTER_KEY_REGEX = re.compile("^[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12}$")
```

Comment:
```suggestion
    RE_MASTER_KEY = re.compile("^[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12}$")
```
I know it wasn't called this before but this is more consistent with other regexes.